### DISCUSSION TITLE
Deploying Hyperledger Fabric blockchain network on Akash Kubernetes cluster

### DISCUSSION THREAD

**Discussion by mushirneo** [akash-network/community#245] [Q&A]
Hyperledger Fabric network requires GRPC communicaton for fabric entities to communicate internally, so is this kind of deployment possible on Akash Kubernetes cluster? Along with that, what other considerations should we keep in mind while deploying our private blockchain network since I am not able to find any documentation regarding it on the official site. If there is anyone who has tried it earlier then please can you share your experience

**Comment by ghost** [comment]
Hi @mushirneo and welcome to the Akash.

As I've read more on Hyperledger Fabric blockchain network https://hyperledger-fabric.readthedocs.io/en/release-2.5/prereqs.html it does appear to be using a standard gRPC.

gRPC uses HTTP/2 under the hood which operates over the TCP protocol.

So it should work.

Akash uses K8s under the hood which does support the following network protocols:
- IPv4, IPv6 (in private testing currently), TCP/IP, ICMP, UDP, HTTP/HTTPS, gRPC, HTTP/2.

Hope that answers your question.

Let us know if you have any further questions.

**Reply by mushirneo** [reply]
Thankyou for your response @andy108369 
But the Hyperledger Fabric network requires inter-pod connection to work but in one of the answers quoted here, it says inter-pod communication is disabled for security purpose - Can you throw some light on this please? - 
https://github.com/orgs/akash-network/discussions/171#discussioncomment-5645941
